### ISSUE TITLE
downloadUrls doesn't exist

### ISSUE THREAD

**Issue by baptistemanson** [chbrown/parallel-wget#1]
Hi,

I couldn't find a reference to downloadUrls in the code... is something missing?

**Comment by chbrown** [comment]
Yeah, I originally used it just as a command line tool and the README got out of date.

For now, until I have time to get back into this project and tidy it up, you're probably better off using `xargs` to parallelize, or draw out your task in a Makefile and use `make -j`.

Sorry I can't be of more immediate help!
